src/context/directory/handlers/organizations.js

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs-extra';
+import path from 'path';
+
+import log from '../../../logger';
+import { getFiles, existsMustBeDir, dumpJSON, loadJSON, sanitize } from '../../../utils';
+
+function parse(context) {
+  const organizationsFolder = path.join(context.filePath, 'organizations');
+
+  if (!existsMustBeDir(organizationsFolder)) return { organizations: undefined }; // Skip
+
+  const files = getFiles(organizationsFolder, [ '.json' ]);
+
+  const organizations = files.map((f) => {
+    const org = { ...loadJSON(f, context.mappings) };
+    return org;
+  });
+
+  return {
+    organizations
+  };
+}
+
+async function dump(context) {
+  const { organizations } = context.assets;
+
+  // API returns an empty object if no grants are present
```

Comment:
> no grants are present
Typo?

Reply:
nice, will fix

Reply:
I don't know why I live comments if I don't read them 😂

Reply:
You know how I love comments :trollface: